pkg/instrumentation/metrics.go

```diff
@@ -132,7 +134,7 @@ func NewMetrics(reg *prometheus.Registry) *Metrics {
 			Namespace: NameSpace,
 			Name:      RHCertExpiryDays,
 			Help:      "Number of days until the Red Hat client certificate expires",
-		}),
+		}), // TODO remove in favor of CertificateExpiry Days
```

Comment:
are you planning on doing that as a later PR after setting up the alerts or is this something you forgot to do? 😅

Reply:
yeah that will come later. the alert for this metric has to be removed first

Reply:
alrighty, thanks!